doc/changelog.rst

```diff
@@ -794,8 +794,14 @@ in this release.
 .. _PYTHON-3186: https://jira.mongodb.org/browse/PYTHON-3186
 .. _pymongo-stubs: https://github.com/mongodb-labs/pymongo-stubs
 
-Changes in Version 4.0
-----------------------
+Changes in Version 4.0.2 (2022/03/03)
+-------------------------------------
```

Comment:
There were no commits in https://github.com/mongodb/mongo-python-driver/compare/v4.0...4.0.2, so I think we should just say no commits due to lack of historical context.